Missing version in inventory.yaml causes an error

## Describe the Bug
If a bolt configuration file is present without a `version` string you get the following stack trace when running `rake`.

```
rake aborted!
Inventory file is incompatible (version 2 and up). Try the 'bolt project migrate' command.
/home/nmaludy/.pdk/cache/ruby/2.5.0/gems/puppet_litmus-0.21.0/lib/puppet_litmus/inventory_manipulation.rb:21:in `inventory_hash_from_inventory_file'
/home/nmaludy/.pdk/cache/ruby/2.5.0/gems/puppet_litmus-0.21.0/lib/puppet_litmus/rake_tasks.rb:340:in `block (2 levels) in <top (required)>'
/home/nmaludy/.pdk/cache/ruby/2.5.0/gems/puppet_litmus-0.21.0/lib/puppet_litmus/rake_tasks.rb:337:in `block in <top (required)>'
/home/nmaludy/.pdk/cache/ruby/2.5.0/gems/puppet_litmus-0.21.0/lib/puppet_litmus/rake_tasks.rb:5:in `<top (required)>'
/home/nmaludy/storage/home/src/xxx/puppet-xxx/Rakefile:3:in `require'
/home/nmaludy/storage/home/src/xxx/puppet-xxx/Rakefile:3:in `<top (required)>'
/opt/puppetlabs/pdk/private/ruby/2.5.8/bin/bundle:23:in `load'
/opt/puppetlabs/pdk/private/ruby/2.5.8/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)
make: *** [.test-lint] Error 1
```

However, the `version` parameter is deprecated in inventory.yaml, so when you add it you get the following warning from Bolt:
```
$ bolt plan run xxx
Unknown option 'version' at /home/nmaludy/storage/home/src/xxx/inventory.yaml.
```

## Expected Behavior

Litmus should not cause errors with `rake`, especially when not using it.

## Steps to Reproduce

1. Install pdk
2. Create a valid `inventory.yaml`
```
---
config:
  transport: ssh
  
groups:
  - name: test
    targets: ['localhost']
```
3. run `pdk bundle exec rake`

## Environment
 - Version `0.21.0`
 - Platform: RHEL 7

--- a/lib/puppet_litmus/inventory_manipulation.rb
+++ b/lib/puppet_litmus/inventory_manipulation.rb
@@ -17,10 +17,7 @@ module PuppetLitmus::InventoryManipulation
                           end
     raise "There is no inventory file at '#{inventory_full_path}'." unless File.exist?(inventory_full_path)
 
-    inventory_hash = YAML.load_file(inventory_full_path)
-    raise "Inventory file is incompatible (version 2 and up). Try the 'bolt project migrate' command." if inventory_hash['version'].nil? || (inventory_hash['version'] < 2)
-
-    inventory_hash
+    YAML.load_file(inventory_full_path)
   end
 
   # Provide a default hash for executing against localhost
@@ -28,7 +25,6 @@ module PuppetLitmus::InventoryManipulation
   # @return [Hash] inventory.yaml hash containing only an entry for localhost
   def localhost_inventory_hash
     {
-      'version' => 2,
       'groups' => [
         {
           'name' => 'local',

